exercise24.py

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu May 10 15:34:22 2018
+
+@author: amyhaddad
+"""
+
+def is_anagram(first_word, second_word):
+    """Test if words are anagrams"""
+
+    if len(first_word) == len(second_word):
```

Comment:
@paulghaddad Is this the best way to write this function? Originally, I started writing a for loop, but it got a bit convoluted.